tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -57,14 +57,14 @@ PDM1_MIC_B_ENABLE=1,DMIC0_PCM_ID=99,PREPROCESS_PLUGINS=nhlt,\
 NHLT_BIN=nhlt-sof-mtl-max98357a-rt5682.bin,DEEPBUFFER_FW_DMA_MS=10,HEADSET_SSP_DAI_INDEX=2,\
 SPK_ID=6,SPEAKER_SSP_DAI_INDEX=0,HEADSET_CODEC_NAME=SSP2-Codec,SPEAKER_CODEC_NAME=SSP0-Codec,\
 BT_NAME=SSP1-BT,BT_INDEX=1,BT_ID=7,BT_PCM_NAME=Bluetooth,INCLUDE_ECHO_REF=true,USE_CHAIN_DMA=true,\
-DEEPBUFFER_D0I3_COMPATIBLE=true,GOOGLE_RTC_AEC_SUPPORT=1"
+DEEPBUFFER_D0I3_COMPATIBLE=true,GOOGLE_RTC_AEC_SUPPORT=1,DEEP_BUF_2=true"
```

Comment:
Long term, we need a better name or flag to say a PCM is deep buffer i.e. ```DEEP_BUFFER_HP=true```. Not a blocker for this PR though, can be looked at later.

Reply:
+1. DEEP_BUFFER_HP=true & DEEP_BUFFER_SPK= true gives better understanding

Reply:
Ack, will update DEEP_BUFFER_2 to DEEP_BUFFER_SPK.
so far we have single deepbuf PCM,  DEEPBUFFER_FW_DMA_MS is used to represents DEEP_BUFFER_HP, can we use a seperate patch to add DEEP_BUFFER_HP since it will touch more files? Thanks,